### DISCUSSION TITLE
Hex color support

### DISCUSSION THREAD

**Discussion by Quackoramic** [GriefPrevention/GriefPrevention#2208] [Ideas]
It would be amazing if GP could support hex colors like this: $#rrggbb
This currently does not work.

Whilst I understand not everyone may need this, my server depends highly on hex colors, and so this feature would be amazing.

**Comment by Jikoo** [comment]
Pretty sure GP translates color codes already, which means you should be able to use Spigot's legacy format of xrrggbb, `&x&r&r&g&g&b&b`.
